plugins/pixart-rf/fu-pxi-device.c

```diff
@@ -254,11 +303,11 @@ fu_pxi_device_reset (FuPxiDevice *self, gsize fw_sz, GError **error)
 	guint8 req[FU_PXI_DEVICE_OTA_BUF_SZ] = {
 		PXI_HID_DEV_OTA_OUTPUT_REPORT_ID,
 		FU_PXI_DEVICE_CMD_FW_MCU_RESET,
-		fw_sz, /* FIXME: this is size_t being sqeezed into uint8_t?! */
+		OTA_RESET,
 	};
 	fu_device_set_status (FU_DEVICE (self), FWUPD_STATUS_DEVICE_RESTART);
 	if (!fu_udev_device_pwrite_full (FU_UDEV_DEVICE (self), 0,
-					 req, sizeof(req),
+					 req, 0x03,
```

Comment:
isn't this the same?

Reply:
Yes ,I want specify the length , if  length over BLE MTU , it could be error